<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>732</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::VideoDisplay"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>732</width>
     <height>300</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>732</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Video to Image Converter!</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="selection_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>711</width>
       <height>131</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="vid_label">
        <property name="text">
         <string>Video path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="vid_button">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="vid_dir">
        <property name="text">
         <string>No video file selected.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="out_label">
        <property name="text">
         <string>Output path</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="out_button">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="out_dir">
        <property name="text">
         <string>No output directory selected.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>711</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Hello ^o^/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="status_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>711</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="loading_bar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="convert_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Convert!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="quality_group">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="quality_label">
        <property name="text">
         <string>Quality :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="low_quality_box">
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="medium_quality_box">
        <property name="text">
         <string>Medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="high_quality_box">
        <property name="text">
         <string>High</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="configuration_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="start_label">
        <property name="text">
         <string>Start :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="start_box"/>
      </item>
      <item>
       <widget class="QLabel" name="end_label">
        <property name="text">
         <string>End :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="end_box"/>
      </item>
      <item>
       <widget class="QLabel" name="frame_label">
        <property name="text">
         <string>Frame Skip : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="frame_box"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
